### PR TITLE
Release 1.3.4

### PR DESCRIPTION
**Pull request by rainest**
**What this PR does / why we need it**:
Release 1.3.4, a 1.3.x with encrypted reporting

**Special notes for your reviewer**:
Some changelog weirdness due to the differences between the new and old formats. I accidentally changed 1.3.3's also but whatever, no meaningful difference to the function.

We should maybe just omit backported changelogs altogether since I don't think anyone is looking at the branch/tagged versions--the complete version in main should be sufficient and the single source of truth for all released versions.

The release CI does not support 1.3.x due to changes in the Dockerfile layout. 1.3.4 will require https://github.com/Kong/kubernetes-ingress-controller/blob/main/RELEASE.md#manual-docker-image-build

Codecov still runs PR checks in this branch and is its usual confusing self. Nothing we can do to make it happy and I don't think it's worth stripping it.

**PR Readiness Checklist**:

See https://github.com/Kong/kubernetes-ingress-controller/pull/2183. Tentatively actually building and releasing tomorrow.

### CODE REVIEW

**Review by mflendrich — COMMENTED** [review]
1 comment, otherwise lgtm

**Comment by mflendrich** [comment]
> Codecov still runs PR checks in this branch and is its usual confusing self. Nothing we can do to make it happy and I don't think it's worth stripping it.

In order to get things right in the first place, I've drafted https://github.com/Kong/kubernetes-ingress-controller/pull/2188 to fix this problem at source. Requires verification that it actually works as intended and possibly cherry-picking into the release PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,7 +32,16 @@
  - [0.0.5](#005---20180602)
  - [0.0.4 and prior](#004-and-prior)
 
-## [1.3.3] - 2021/10/01
+## [1.3.4]
```
[the remark this answers is not in the document]
Comment (rainest) [reply CHANGELOG.md]: Now removed entirely, updates only in main.

**Review by shaneutt — APPROVED** [review]

**Comment by shaneutt** [comment]
We discussed in a call that for this special occasion we're OK with merging through the codecov failure because it's not really applicable, LGTM :+1:

**Pull request merged**